.circleci/config.yml

```diff
@@ -1,165 +1,44 @@
-          command: npm ci
-      - persist_to_workspace:
-          root: ~/
-          paths:
-            - project
-
-  build-distributable:
-    docker:
-      - image: cimg/php:7.4-browsers
-    steps:
-      - checkout_with_workspace
-      - run:
-          name: Install rsync
-          command: sudo apt-get update && sudo apt-get install rsync
-      - run:
-          name: Install PHP packages
-          command: composer install --no-dev --no-scripts
-      - run:
-          name: Build plugin files
-          command: npm run build && npm run release:archive
-      - store_artifacts:
-          path: release/newspack-plugin.zip
-
-  # JS & SCSS Jobs
-  lint-js-scss:
-    docker:
-      - image: cimg/php:7.4-browsers
-    steps:
-      - checkout_with_workspace
-      - run:
-          name: Run Linter
-          command: npm run lint
-
-  test-js:
-    docker:
-      - image: cimg/php:7.4-browsers
-    steps:
-      - checkout_with_workspace
-      - run:
-          name: Run JS Tests
-          command: npm run test
-
-  # PHP jobs
-  lint-php:
-    docker:
-      - image: cimg/php:7.4
-    steps:
-      - checkout
-      - run:
-          name: Lint Files
-          command: |
-            composer install
-            ./vendor/bin/phpcs
-
-  test-php:
-    docker:
-      - image: cimg/php:7.4
-      - image: circleci/mysql:5.6.50
-    environment:
-      - WP_TESTS_DIR: '/tmp/wordpress-tests-lib'
-      - WP_CORE_DIR: '/tmp/wordpress/'
-    steps:
-      - checkout
-      - run:
-          name: Setup Environment Variables
-          command: |
-            echo "export PATH=$HOME/.composer/vendor/bin:$PATH" >> $BASH_ENV
-            source /home/circleci/.bashrc
-      - run:
-          name: Install Dependencies
-          command: |
-            sudo apt-get update && sudo apt-get install -y subversion default-mysql-client
-      - run:
-          name: Run Tests
-          command: |
-            composer install
-            composer global require "phpunit/phpunit=7.5.*"
-            rm -rf $WP_TESTS_DIR $WP_CORE_DIR
-            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 5.8.1
-            phpunit
-
-  # Release job
-  release:
-    docker:
-      - image: cimg/php:7.4-browsers
-    steps:
-      - checkout_with_workspace
-      - run:
-          name: Install rsync
-          command: sudo apt-get update && sudo apt-get install rsync
-      - run:
-          name: Install PHP packages
-          command: composer install --no-dev --no-scripts
-      - run:
-          name: Release new version
-          command: npm run release
-      - persist_to_workspace:
-          root: ~/
-          paths:
-            - project
-
-  post_release:
-    docker:
-      - image: cimg/php:7.4-browsers
-    steps:
-      - checkout_with_workspace
-      - run:
-          name: Perform post-release chores
-          command: ./node_modules/newspack-scripts/post-release.sh
+orbs:
+  newspack: adekbadek/newspack@1.1.1
```

Comment:
On CircleCI, [only organisation admins can publish organisation orbs](https://discuss.circleci.com/t/org-users-cannot-publish-orb/26341). I don't think it's worth it here – and the orb code is open-source at `newspack-scripts`, so it can be re-published anytime under a new name [if need be](https://en.wikipedia.org/wiki/Bus_factor).